### PR TITLE
feat: add FAQ section

### PR DESCRIPTION
**Pull request by jbrockopp**
This adds a Frequently Asked Questions (F.A.Q.) section to the docs for Vela.

The content is primarily focused around common questions that surface around usage of the product for end users.

### CODE REVIEW

**Review by JordanSussman — DISMISSED** [review]

File: content/faq/usage/_index.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+
+![Build Is Pending](build_is_pending.png)
+
+This behavior indicates the number of `running` builds for the system is greater than the number of [workers](/docs/administration/worker/) available.
+
+Unfortunately, the only way to resolve the issue is to wait until a worker becomes available to run your build.
+
+{{% alert title="Tip:" color="primary" %}}
+Reach out to the administrators of the installation to request additional workers be added to the system.
+{{% /alert %}}
```
Comment (wass3r) [inline content/faq/usage/_index.md]: I think this should be taken out. It's making some assumptions about the flexibility of an installation.

**Review by wass3r — CHANGES_REQUESTED** [review]
thanks - love the addition. one change i would propose. i'm always hesitant for including screenshots as well. case in point, they are already outdated (albeit not majorly different at this point). not a request for changing it, more of a general statement.

**Comment by jbrockopp** [comment]
@JordanSussman @wass3r this should be RFR 👍

File: content/faq/usage/_index.md (hunk 1 above)
Comment (jbrockopp) [reply content/faq/usage/_index.md]: this is fixed 👍

**Review by wass3r — APPROVED** [review]
thanks!

**Review by JordanSussman — APPROVED** [review]

**Pull request merged**